packages/react-storage/src/components/StorageBrowser/components/DropdownMenu.tsx

```diff
@@ -63,7 +63,11 @@ export function DropdownMenu({
       <ButtonElement
         aria-label="Menu Toggle"
         className={`${STORAGE_BROWSER_BLOCK_TO_BE_UPDATED}__menu-toggle`}
-        disabled={isDisabled || !items?.length}
+        disabled={
+          isDisabled ||
+          !items?.length ||
+          items?.filter((obj) => obj.isHidden).length == items?.length
```

Comment:
May be better to check if `actions` are empty [here](https://github.com/aws-amplify/amplify-ui/blob/9385d5831602f6fb31964090324b0e8d9f780bbd/packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx#L71)

Reply:
yeah, will move the check logic there